jsapp/js/components/processing/sidebar/sidebarSubmissionData.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from 'react';
```

Comment:
This file needs prettier to be run over it :) Make sure you re-run prettier over all the files you touch just to be sure